CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Change Log
 
+## [0.23.0](https://github.com/eclipse/lemminx/milestone/36?closed=1) (December 15, 2022)
+
+### Enhancements
+
+ * Codelens, completion, definition, documentLink, highlighting, references, rename support in RNG file. See [#1400](https://github.com/eclipse/lemminx/pull/1400).
+ * Surround selection with XML element. See [#1389](https://github.com/eclipse/lemminx/pull/1389).
+ * Switch experimental formatter to default. See [#1377](https://github.com/eclipse/lemminx/issues/1377).
+ * Full support for `xml.format.maxLineWidth` with experimental formatter. See [#1248](https://github.com/eclipse/lemminx/issues/1248).
+ * Adjust selection range for wrap element when empty selection is in start or end tag. See [#1392](https://github.com/eclipse/lemminx/pull/1392).
+ * Support `files.trimTrailingWhitespace` setting with experimental formatter. See [#1310](https://github.com/eclipse/lemminx/issues/1310).
+ * Remove `xml.format.preserveEmptyContent` for experimental formatter. See [#1346](https://github.com/eclipse/lemminx/issues/1346).
+ * Support `xml.format.maxLineWidth` with invalid attribute values. See [#1363](https://github.com/eclipse/lemminx/pull/1363).
+ * Extend `xml.format.maxLineWidth` setting to support attributes and tag elements. See [#1359](https://github.com/eclipse/lemminx/pull/1359).
```

Comment:
I would remove these 2 since they are basically implied by "Full support for `xml.format.maxLineWidth` with experimental formatter". If you want, you could maybe list their issue numbers after the "See .."